endpoints/categories/category.php

```diff
@@ -1,9 +1,16 @@
 <?php
 require_once '../../includes/connect_endpoint.php';
 session_start();
+function validate($value) {
+    $value = trim($value);
+    $value = stripslashes($value);
+    $value = htmlspecialchars($value);
+    $value = htmlentities($value);
+    return $value;
+}
 if (isset($_SESSION['loggedin']) && $_SESSION['loggedin'] === true) {
     if (isset($_GET['action']) && $_GET['action'] == "add") {
-        $categoryName = "Category";
+        $categoryName = validate("Category");

```

Comment:
This value "Category" is hardcoded and not a user input, no need for validation here.